spread/build/debian/task.yaml

```diff
@@ -0,0 +1,34 @@
+summary: Build (on Debian)
+systems: [debian-*]
+
+execute: |
+    # Grab builds of Mir git master
+    add-apt-repository ppa:mir-team/dev
```

Comment:
*Pretty* sure this will only work properly on Ubuntu. We might just need to build against the latest Mir release in Debian?

Reply:
That looks to be 1.8: https://buildd.debian.org/status/package.php?p=mir

But why would the wlcs build depend on Mir? (I know we do it on our infrastructure, but it shouldn't be a hard dependency anywhere else.)